kuma/javascript/src/signin-link.jsx

```diff
@@ -0,0 +1,70 @@
+//@flow
+import * as React from 'react';
+import { useContext } from 'react';
+
+import { getLocale, gettext } from './l10n.js';
+import GAProvider from './ga-provider.jsx';
+
+type Props = {
+    className?: string,
+    text?: string,
+};
+
+export default function SignInLink({
+    className,
+    text = gettext('Sign in'),
+}: Props): React.Node {
+    const ga = useContext(GAProvider.context);
+    const locale = getLocale();
+
+    if (typeof window === 'undefined') {
```

Comment:
I believe we used to do this test this way because in the past you could do:

`undefined = "monkey-wrench";` or even `undefined = "";`

In strict mode(which we are in) `undefined` is immutable(non-writable global) and so one can simply do:

```
if (window === undefined)
```

Reply:
Good to know! I ended up removing this altogether per Peter's comment below.